images/scripts/update-hubble-version.sh

```diff
@@ -34,10 +36,13 @@ for arch in amd64 arm64 ; do
   hubble_sha256+=("${sha256[0]}")
 done
 
-cat > "${root_dir}/images/cilium/hubble-version.sh" << EOF
+cat > "$hubble_version_script" << EOF
 # Code generated by images/scripts/update-hubble-version.sh; DO NOT EDIT.
 hubble_version="${hubble_version}"
 declare -A hubble_sha256
 hubble_sha256[amd64]="${hubble_sha256[0]}"
 hubble_sha256[arm64]="${hubble_sha256[1]}"
 EOF
+
+sed -e "s/\\(image: .*\\/hubble:\\).*/\1${hubble_version}/" "$hubble_cli_example" > "${hubble_cli_example}.tmp" &&
```

Comment:
tiny nit (a personal preference to get rid of the `\\/`, feel free to ignore):

```suggestion
sed -e "s#\\(image: .*/hubble:\\).*#\1${hubble_version}#" "$hubble_cli_example" > "${hubble_cli_example}.tmp" &&
```